sdk/apimanagement/api-management-custom-widgets-scaffolder/package.json

```diff
@@ -41,7 +41,7 @@
     "extract-api": "tsc -p . && dev-tool run extract-api",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore --ignore-path ./.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "dev-tool run test:vitest --no-test-proxy -- -coverage",
+    "integration-test:node": "dev-tool run test:vitest --no-test-proxy -- --coverage",
```

Comment:
just curious why only these packages have been impacted?

Reply:
I am working on core first.  Yes, I'd expect generated packages to have similar changes to enable code coverage reports in live test runs.

Reply:
Got it and thanks for the info.